src/finch/types/connect/session_new_params.py

```
@@ -5,7 +5,7 @@
 from typing import List, Optional
 from typing_extensions import Literal, Required, TypedDict
 
-__all__ = ["SessionNewParams", "Integration"]
+__all__ = ["SessionNewParams", "Integration", "Recordkeeping"]
```

Ruff (0.16.1)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)